docs/src/pages/docs/crocks/Pair.md

```diff
@@ -887,14 +887,12 @@ const record = {
 const joinField = (key, value) =>
   `${key}:${value}`
 
-// joinRecord :: List String -> String
-const joinRecord = list =>
-  list.toArray()
-    .join('|')
+// joinRecord :: String -> [String] -> String
+const join = separator => array => array.join(separator)
```

Comment:
`array.join(separator)` should be on a second line

Reply:
I believe we could also simplify this to `const join = array => array.join`

Reply:
done.